helm-charts/eap72/templates/tests/pod.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ .Release.Name }}-test
+  annotations:
+    "helm.sh/hook": test-success
```

Comment:
Since we have to manually start a build for the application to eventually roll out, this would be hard to automate

Reply:
Now that the "ConfigChange" trigger is added to the BuildConfig, testing should be simpler.

Reply:
Agree